src/app/shared/components/text-input/text-input.component.html

```diff
@@ -1,21 +1,15 @@
 <ng-container class="form-group">
-  <input #customInput
-         id="customInput"
-         [type]="type"
-         [name]="name"
-         ngModel
-         class="form-control"
-         autocomplete="off"
-         required>
+  <input [(ngModel)]="value" class="form-control" type="{{type}}">
```

Comment:
When passing down data as @Input, it's better to use binding, instead of interpolation.

Reply:
ok